tools/genevergreen/generate/generate.go

```diff
@@ -181,7 +181,7 @@ func PostPkgMetaTasks(c *shrub.Configuration, toolName string) {
 func PublishStableTasks(c *shrub.Configuration, toolName string) {
 	dependency := []shrub.TaskDependency{
 		{
-			Name:    fmt.Sprintf("compile_%s", toolName),
+			Name:    "compile",
```

Comment:
Why was toolName causing the issue? you still have the toolName as input for the function.

Reply:
The `compile_mongocli` task was removed. The `compile` task was left in the mongocli branch.